### PR TITLE
Removing Path, WellKnownFeatures from Dependency.proto

### PR DESCRIPTION
**Pull request by SatishRanjan**

### CODE REVIEW

File: src/proto/FunctionRpc.proto [hunk 1]
```diff
@@ -207,8 +204,8 @@ message FunctionLoadRequest {
   // Metadata for the request
   RpcFunctionMetadata metadata = 2;
 
-  //Used to send the dependency management request to the language worker
-  DownloadDependencyRequest download_dependency_request = 3;
+  //Used to send the dependency download request to the language worker
+  Dependency dependency_download_request = 3;
```
Comment (pragnagopa) [inline src/proto/FunctionRpc.proto]: change to bool managed_dependency_enabled

File: src/proto/ManagedDependency/Dependency.proto [hunk 2]
```diff
@@ -9,22 +9,9 @@ option go_package ="github.com/Azure/azure-functions-go-worker/internal/rpc";
...
-
-  //The path where managed dependency modules will be installed e.g. '/home/manageddependencies'
-  string path = 3;
-}
-
-//Used to send the dependency download request to the language worker
-message DownloadDependencyRequest {
-  //The repeated list of dependencies
-  repeated Dependency dependencies = 1;
+  //Flag indicating if managed dependency is enabled or not
```
Comment (pragnagopa) [inline src/proto/ManagedDependency/Dependency.proto]: delete. For each function, sending managed depency enabled or not is enough

**Comment by pragnagopa** [comment]
Ping! Commit history does not look right. Please rebase

File: src/proto/ManagedDependency/Dependency.proto (hunk 2 above)
Comment (SatishRanjan) [reply src/proto/ManagedDependency/Dependency.proto]: Resolved

**Pull request closed without merging**